### PR TITLE
Review request: Members only project

### PR DESCRIPTION
**Pull request by ahmetbozaci**
## In these projects, we  worked to implement authentication systems
- We used devise gem for authentication handling.

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @ahmetbozaci and @chrystalme  :wave: 

You've done well so far :+1:  and I commend you for that :clap: 
However, there are still some changes to be made before this project can be approved.

## REQUIRED

#### Application layout

* [ ] The application layout has Sign in / Sign out links (So they are not manually repeated in every view) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_out_link_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49722686/))

### Make the required change and request another review.
## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
# **Changes Required 🛑♻**

<hr>

Hello Team, great job working on this project and meeting all of its specifications. However, there is one professional issue and another technical one that needs to be fixed for best practices.

- [ ] Kindly follow the Git Flow Guidelines and merge to develop branch in the PR.
- [ ] Please move any related logic code from the views into the helpers as ruby methods, then call them back where you need in the views (There should not be any logic code in the views).

<hr>

Kindly do the changes and ask for another review.
Another hint: make sure to always update your .gitignore file with the large files like vendor/bundle and node_modules before pushing to Github because currently there are more than 5k changed files in this PR.

Good luck and happy coding 👨‍💻💻

<hr>

You can reach me on Slack with the name "Mahmoud Bakr".

**Pull request closed without merging**
